src/chainparams.cpp

```diff
@@ -157,8 +157,8 @@ class CMainParams : public CChainParams {
         vSeeds.emplace_back("seed.bitcoin.jonasschnelli.ch", true); // Jonas Schnelli, only supports x1, x5, x9, and xd
         vSeeds.emplace_back("seed.btc.petertodd.org", true); // Peter Todd, only supports x1, x5, x9, and xd
 
-        base58Prefixes[PUBKEY_ADDRESS] = std::vector<unsigned char>(1,0);
-        base58Prefixes[SCRIPT_ADDRESS] = std::vector<unsigned char>(1,5);
+        base58Prefixes[PUBKEY_ADDRESS] = std::vector<unsigned char>(1,38);  // prefix: G
+        base58Prefixes[SCRIPT_ADDRESS] = std::vector<unsigned char>(1,90);  // prefix: d
```

Comment:
Better to use both capital letters

Reply:
Actually, as was offered by h4x3rotab, **G** and **g** could be used, anyone who has met bitcoin notices that the address is written in different registers, not a big deal if it will be same letter, because users copy-paste them. Many users still didn't understand difference between 1 and 3. Maybe you can consider **A** (decimal -> 23) as the title referenced to Aurum (gold in periodic table of element).

Reply:
Changed to "G" and "A"